apps/projects/helpers.py

```diff
@@ -11,24 +11,7 @@
 
 def get_all_comments_project(project):
     return Comment.objects.filter(
-        Q(budget_proposal__module__project=project)
```

Comment:
the individual filtering here is really slow, do we really need it? what other comments are there which we don't want to show here?